### ISSUE TITLE
Spyder within Python 2.7 Anaconda environment crashes

### ISSUE THREAD

**Issue by mfogle** [spyder-ide/spyder#13962]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [x] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [x] Reset preferences with ``spyder --reset``
    * [x] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [x] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
Platform - HP Envy, 10th Gen Intel i7, Windows 10 Pro 1909 64-bit.

After from-scratch reinstall of Anaconda3 1.9.12 Navigator (in hopes of clearing the Spyder IDE issues), created a Python 2.7 virtual environment using Anaconda Cmd.exe (conda) prompt.  Installed both Spyder 4.1.4 (default) as well as rolled back to Spyder 3.2.8 and 3.2.3 (earliest I can go) - all have same behavior within the Python 2.7 virtual environment - Spyder fails to start and get the dialog box shown in the attachment below.

![Spyder_3 2 8Crash_Screenshot](https://user-images.githubusercontent.com/6884879/95688164-0df50380-0bd6-11eb-8510-8631e72df4f2.png)

From the base Python environment (Python 3.8.3) Spyder 4.1.4 (default version) can be successfully launched.

### What steps reproduce the problem?

1. Download and install latest Anaconda3 from https://www.anaconda.com/products/individual
2. Launch Navigator
3. Launch conda Cmd.exe
4. "conda create -n py2_7 python=2.7"
5. Exit Anaconda3 Navigator, then re-execute.
6. Switch to "py2_7" via "Applications on" pull-down within Navigator.
7. Install Spyder (tried default version 4.1.4, 3.2.8, and 3.2.3)
8. Launch Spyder (any of the versions mentioned in step 7) - get aforementioned dialog, i.e. "Spyder crashed during last session".

Things tried but issue still persists.
1. "spyder --reset".
2. Removal of non-base environments via "conda remove" in CMD.exe Prompt, then reinstalling Anaconda from scratch.
3. From CMD.exe Prompt, uninstall then reinstall Spyder ("conda uninstall" and "conda install".  Uninstall and re-install are without error, but Spyder still crashes.
4. After creating the "py2_7" environment, "conda activate py2_7" then run Spyder from command line.  Get traceback shown in traceback section.

### What is the expected output? What do you see instead?
I expect to see the Spyder IDE open.  What I see instead is either:
1. (execution from Navigator) Error dialog previously attached.
2. (execution from CMD.exe conda console) Traceback as shown below.

### Paste Traceback/Error Below (if applicable)

```python-traceback

Microsoft Windows [Version 10.0.18363.1082]
(c) 2019 Microsoft Corporation. All rights reserved.

(py2_7) C:\Users\micha>spyder --show-console
(Deprecated) --show console does nothing, now the default  behavior is to show the console, use --hide-console if you want to hide it
Traceback (most recent call last):
  File "C:\Users\micha\anaconda3\envs\py2_7\lib\site-packages\spyder\app\mainwindow.py", line 3208, in main
    mainwindow = run_spyder(app, options, args)
  File "C:\Users\micha\anaconda3\envs\py2_7\lib\site-packages\spyder\app\mainwindow.py", line 3082, in run_spyder
    main = MainWindow(options)
  File "C:\Users\micha\anaconda3\envs\py2_7\lib\site-packages\spyder\app\mainwindow.py", line 373, in __init__
    from spyder.plugins.shortcuts import ShortcutsConfigPage
  File "C:\Users\micha\anaconda3\envs\py2_7\lib\site-packages\spyder\plugins\shortcuts.py", line 835, in <module>
    class ShortcutsConfigPage(GeneralConfigPage):
  File "C:\Users\micha\anaconda3\envs\py2_7\lib\site-packages\spyder\plugins\shortcuts.py", line 839, in ShortcutsConfigPage
    ICON = ima.icon('keyboard')
  File "C:\Users\micha\anaconda3\envs\py2_7\lib\site-packages\spyder\utils\icon_manager.py", line 267, in icon
    directory=_resource['directory'])
  File "C:\Users\micha\anaconda3\envs\py2_7\lib\site-packages\qtawesome\__init__.py", line 178, in load_font
    return _instance().load_font(prefix, ttf_filename, charmap_filename, directory)
  File "C:\Users\micha\anaconda3\envs\py2_7\lib\site-packages\qtawesome\__init__.py", line 54, in _instance
    'materialdesignicons-webfont-charmap.json')
  File "C:\Users\micha\anaconda3\envs\py2_7\lib\site-packages\qtawesome\iconic_font.py", line 206, in __init__
    super().__init__()
TypeError: super() takes at least 1 argument (0 given)

(py2_7) C:\Users\micha>spyder --reset
*** Reset Spyder settings to defaults ***
removing: C:\Users\micha\.spyder\spyder.ini
removing: C:\Users\micha\.spyder\langconfig
removing: C:\Users\micha\.spyder\spyder.lock

(py2_7) C:\Users\micha>spyder --show-console
(Deprecated) --show console does nothing, now the default  behavior is to show the console, use --hide-console if you want to hide it
Traceback (most recent call last):
  File "C:\Users\micha\anaconda3\envs\py2_7\lib\site-packages\spyder\app\mainwindow.py", line 3208, in main
    mainwindow = run_spyder(app, options, args)
  File "C:\Users\micha\anaconda3\envs\py2_7\lib\site-packages\spyder\app\mainwindow.py", line 3082, in run_spyder
    main = MainWindow(options)
  File "C:\Users\micha\anaconda3\envs\py2_7\lib\site-packages\spyder\app\mainwindow.py", line 373, in __init__
    from spyder.plugins.shortcuts import ShortcutsConfigPage
  File "C:\Users\micha\anaconda3\envs\py2_7\lib\site-packages\spyder\plugins\shortcuts.py", line 835, in <module>
    class ShortcutsConfigPage(GeneralConfigPage):
  File "C:\Users\micha\anaconda3\envs\py2_7\lib\site-packages\spyder\plugins\shortcuts.py", line 839, in ShortcutsConfigPage
    ICON = ima.icon('keyboard')
  File "C:\Users\micha\anaconda3\envs\py2_7\lib\site-packages\spyder\utils\icon_manager.py", line 267, in icon
    directory=_resource['directory'])
  File "C:\Users\micha\anaconda3\envs\py2_7\lib\site-packages\qtawesome\__init__.py", line 178, in load_font
    return _instance().load_font(prefix, ttf_filename, charmap_filename, directory)
  File "C:\Users\micha\anaconda3\envs\py2_7\lib\site-packages\qtawesome\__init__.py", line 54, in _instance
    'materialdesignicons-webfont-charmap.json')
  File "C:\Users\micha\anaconda3\envs\py2_7\lib\site-packages\qtawesome\iconic_font.py", line 206, in __init__
    super().__init__()
TypeError: super() takes at least 1 argument (0 given)


```

## Versions

* Spyder version: 4.1.4, 3.2.8, 3.2.3
* Python version: 2.7.18
* Qt version: 
* PyQt version: 
* Operating System name/version: Windows 10 Pro x64 1909 

### Dependencies

```
Spyder crashes in py2_7 environment so can't get to the Dependencies listing.
```

**Comment by steff456** [comment]
Hi @mfogle,

Spyder 4.1.4 doesn't support Python 2 anymore, we highly recommend to move all your code base to python 3 as python 2 lost support since December of last's year. If you still need to run code in python please check the thread in the issue #13510, hope this works for you!

**Comment by ccordoba12** [comment]
> File "C:\Users\micha\anaconda3\envs\py2_7\lib\site-packages\qtawesome\iconic_font.py", line 206, in __init__
    super().__init__()

Hey @mfogle, thanks for reporting. You need to install QtAwesome 0.7.3 to fix this error for now, but as @steff456 said, it's harder and harder for us to support Python 2. So please consider to move to Python 3 as soon as possible.

**Comment by breedlun** [comment]
Thanks, @ccordoba12, for pointing out the QtAwesome 0.7.3 work around.  I am in the process of finally migrating over to Python 3 and much of my old code is not working (as one might expect), so it is helpful to go back and see how my code behaved under Python 2 before deciding on an approach to resolve a given issue.

**Comment by ccordoba12** [comment]
Sure, no problem. We'll release a last version (4.1.6) with this and all our other dependencies constrained for Python 2, so it can be used in the future.

It won't be possible to install it with conda because Anaconda doesn't create Python 2 packages anymore, but at least it'll be possible to install it with pip (after installing PyQt5 with conda).

**Comment by dalthviz** [comment]
This was fix by #14169

**Comment by XingLiangLondon** [comment]
Downgrade Spyder to version 3.3.6 works for me (From Anaconda Navigator ->Home->Spyder, click Install specific version and choose 3.3.6)
